contracts/deployments/7683/v0.5.0.json

```diff
@@ -0,0 +1,51 @@
+{
+  "metadata": {
+    "version": "v0.5.0",
+    "date": "2025-06-06",
+    "by": "Evan",
```

Comment:
I didn't deploy this. please fix. and please confirm or deny you deployed from branch `feature/deploy-messengers-to-arb-and-base`.

Reply:
Ok I thought we should keep those since a lot of the address come from the previous deployment of 0.4
I deployed from canary branch